src/plugin.js

```diff
@@ -321,19 +321,11 @@ class VR extends Plugin {
   }
 
   requestAnimationFrame(fn) {
-    if (this.vrDisplay) {
-      return this.vrDisplay.requestAnimationFrame(fn);
-    }
-
-    return Component.prototype.requestAnimationFrame.call(this, fn);
+    return this.player_.requestAnimationFrame(fn);
   }
 
   cancelAnimationFrame(id) {
-    if (this.vrDisplay) {
-      return this.vrDisplay.cancelAnimationFrame(id);
-    }
-
-    return Component.prototype.cancelAnimationFrame.call(this, id);
+    return this.player_.cancelAnimationFrame(id);
```

Comment:
I also noticed that we should just call player functions here, as we were getting some un-important errors, but errors none the less.